src/main/java/com/gdsc/studyjamapi/api/UserApi.java

```diff
@@ -92,4 +93,9 @@ ResponseEntity<UserResponse> getUserById(
       })
   @GetMapping("/current")
   ResponseEntity<UserResponse> getCurrentUser();
+  @PutMapping("/{id}")
```

Comment:
You should place a blank line between the two functions.

Reply:
Thank you, let me fix it!